zerver/views/__init__.py

```diff
@@ -632,8 +646,8 @@ def logout_then_login(request, **kwargs):
     # type: (HttpRequest, **Any) -> HttpResponse
     return django_logout_then_login(request, kwargs)
 
-def create_preregistration_user(email, request):
-    # type: (text_type, HttpRequest) -> HttpResponse
+def create_preregistration_user(email, request, create_realm=False):
+    # type: (text_type, HttpRequest, Optional[bool]) -> HttpResponse
```

Comment:
I think `create_realm` should not be tagged as `Optional` -- `Optional` refers to whether the value might be None, not whether it's an optional kwarg.